scripts/perf/clone_database/clone_database.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -euo pipefail
+
+VAULT_TOKEN=$(cat /etc/vault-token-dsde)
+
+typeset DOCKER_ETC_PATH=/usr/share/etc
+
+mkdir -p mnt
+
+# Read the DB password from vault:
+DB_PASS=$( docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN \
+	broadinstitute/dsde-toolbox vault read -format=json secret/dsp/cromwell/perf \
+	| jq --exit-status '.data.db_pass' )
+
+# Read the service account credentials from vault:
+docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN broadinstitute/dsde-toolbox vault read -format=json secret/dsp/cromwell/perf/service-account-deployer | jq --exit-status -r '.data.service_account' > mnt/sa.json
```

Comment:
👍on `jq -r --exit-status '.data.service_account'`, or better yet DRY this file out by moving common functions into an include like `helper.inc.sh`.